src/main.js

```diff
@@ -571,10 +562,40 @@ function initializeAfterAppReady() {
 
   ipcMain.emit('update-dict');
 
-  const permissionFile = path.join(app.getPath('userData'), 'permission.json');
-  const trustedURLs = config.teams.map((team) => team.url);
-  permissionManager = new PermissionManager(permissionFile, trustedURLs);
-  session.defaultSession.setPermissionRequestHandler(permissionRequestHandler(mainWindow, permissionManager));
+  // supported permission types
+  const supportedPermissionTypes = [
+    'media',
+    'geolocation',
+    'notifications',
+    'fullscreen',
+    'openExternal',
+  ];
+
+  // handle permission requests
+  // - approve if a supported permission type and the request comes from the renderer or one of the defined servers
+  session.defaultSession.setPermissionRequestHandler((webContents, permission, callback) => {
+    // is the requested permission type supported?
+    if (!supportedPermissionTypes.includes(permission)) {
+      callback(false);
+      return;
+    }
+
+    // is the request coming from the renderer?
+    if (webContents.id === mainWindow.webContents.id) {
+      callback(true);
+      return;
+    }
+
+    // get the requesting webContents url
+    const requestingURL = webContents.getURL();
+
+    // is the target url trusted?
+    config.teams.forEach((team) => {
+      if (requestingURL.startsWith(team.url)) {
+        callback(true);
+      }
+    });
```

Comment:
what happens if it doesn't belong to any team?

Reply:
hmmm, interesting, I had `callback(false)` in there at one point. Must have inadvertently removed while cleaning up. Good catch!